doc/api/readline.markdown

```diff
@@ -118,7 +118,7 @@ the `input` stream receives a `^C`, respectively known as `SIGINT`.
 
 `function (line) {}`
 
-Emitted whenever the `input` stream receives a `\n`, usually received when the
+Emitted whenever the `input` stream receives `EOL`, usually received when the
```

Comment:
This could be clearer if we explicitly said "end of line (`\n`, `\r`, or `\r\n`)"